src/style.css

```diff
@@ -1,3 +1,97 @@
-body {
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+}
+
+.menu {
+  width: 100%;
+  background-color: var(--background-color);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.logo {
+  margin: 20px;
+  max-height: 50px;
+}
+
+.menu_box {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.menu_link {
+  line-height: auto;
+  font-weight: 500;
+  text-decoration: none;
+  list-style: none;
+  gap: 20px;
+  margin: 20px;
+  font-size: var(--font-size);
+  color: #000;
+  text-transform: uppercase;
+}
+
+.menu_box:first-child {
+  margin-left: 0;
+}
+
+.menu_box:last-child {
+  margin-right: 0;
+}
+
+.menu_link:hover {
+  padding-bottom: 20px;
+  color: var(--main-color);
+  line-height: 10px;
```

Comment:
The `line-height` in the hover state should be consistent with the normal state to prevent layout shifts during interaction. Choose a value that fits the design and apply it in both states.